google/cloud/iam/samples/grpc_credential_types.cc

```diff
@@ -0,0 +1,181 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/iam/iam_credentials_client.gcpcxx.pb.h"
+#include "google/cloud/spanner/instance_admin_client.h"
+#include "google/cloud/internal/getenv.h"
+#include "google/cloud/log.h"
+#include "google/cloud/testing_util/example_driver.h"
+#include "absl/strings/str_split.h"
+#include "absl/time/time.h"  // NOLINT
```

Comment:
Q.  Do you recall what required the `NOLINT`?

Reply:
I think it is this:

https://clang.llvm.org/extra/clang-tidy/checks/modernize-deprecated-headers.html

My guess is that clang-tidy is wrong here (maybe a regular expression that is accepting too much?). Separately, maybe my IDE settings are more strict that necessary, i.e., our builds may not yet care about this particular clang-tidy warning.

Despite all that, I am inclined to leave the NOLINT.